scripts/nesi_scripts/adv_im/run_adv_im.sh

```diff
@@ -0,0 +1,203 @@
+WCT_BENCHMARK_MINSEC=1477 # seconds used to run a ATC12 model on single waveform
+WCT_BENCHMARK_MAXHOUR=6 # the wct for ATC12
+BENCHMARK_OBS_SIZE=30 # the size for submitting obs
+SPLIT_LINE='#####################################'
+
+#############  some tests before running anything ###############
+# quick test to see if model exist
+ADV_IM_MODEL_DIR=$gmsim/IM_calculation/IM_calculation/Advanced_IM/Models/$ADV_IM_NAME
+if [[ ! -d $ADV_IM_MODEL_DIR ]];then
+    echo "directory for adv_im model: $ADV_IM_NAME does not exits"
+    exit 2
+fi
+# check if BBbin_root_dir exist
+if [[ ! -d $BBbin_root_dir ]];then
+    echo "directory for BBbin_root_dir: $BBbin_root_dir does not exits"
+    exit 2
+fi
+
+# check if OBS_DATA_DIR exist
+if [[ ! -d $OBS_DATA_DIR ]];then
+    echo "directory for OBS_DATA_DIR: $OBS_DATA_DIR does not exits"
+    exit 2
+fi
+
+if [[ ! -f $ADV_IM_MODEL_DIR/run.py ]];then
+    echo "run.py for $ADV_IM_NAME cannot be found at $ADV_IM_MODEL_DIR"
+    exit 2
+fi
+
+STATION_LIST=`python -c "from qcore.utils import load_yaml,DotDictify; p=DotDictify(load_yaml(\"$validation_root_params_yaml\"));print(p.stat_file)"`
+
+if [[ ! -f $STATION_LIST ]];then
+    echo "station list for validation run cannot be found on $STATION_LIST"
+    exit 2
+fi
+
+
+USERNAME=`whoami`
+root_dir=$nobackup/RunFolder/$USERNAME/$VALIDATION_VERSION'_'$ADV_IM_NAME
+TASK_CONFIG=$root_dir/task_config.yaml
+# echo $root_dir
+# test if folder name exist
+# if exist, exit
+if [[ -e $root_dir ]];then
+    echo "$root_dir already exist, please delete the old simulation or rename the folder"
+    exit 3
+else
+    mkdir -p $root_dir
+fi
+
+#################################################################
+# OBS related folders
+
+obs_linked_folder=$root_dir/ObservedGroundMotions
+if [[ -e $obs_linked_folder ]];then
+    echo "$obs_linked_folder exist, please remove/rename the old folder"
+    exit 3
+fi
+
+obs_input_dir=$root_dir/obs_input_tmp
+if [[ -e $obs_input_dir ]];then
+    echo "$obs_input_dir exist, please remove/rename the old folder"
+    exit 3
+else
+    mkdir -p $root_dir/obs_input_tmp
+fi
+
+################################################################
+
+# link Data folder
+echo $SPLIT_LINE
+echo "linking Data folder"
+ln -s $SIM_DATA_DIR $root_dir
+
+################################################################
+#   determine mag catagory
+################################################################
+max_mag=`grep mag $root_dir/Data/VMs/*/vm_params.yaml | awk '{print $2}' | sort -unr | head -n 1`
+if (( $(echo "$max_mag > 7.0" | bc -l) ));then
+    mag_category=large
+elif (( $(echo "$max_mag > 5.0" | bc -l) ));then
+    mag_category=moderate
+else
+    mag_category=small
+fi
+
+# install the simulation
+echo $SPLIT_LINE
+echo "installing simulation folder"
+echo $SPLIT_LINE
+python $gmsim/workflow/scripts/cybershake/install_cybershake.py $root_dir $LIST_EVENTS_F $gmsim_version --stat_file_path $STATION_LIST
```

Comment:
the workflow is bacially copying/linking the previous ran BB.bin into a new location, so it will not affect anything in previous runs. and in order to make our automated script submit the jobs, we need to make sure the mgmt_db have all the steps and in clean-state . And i think creating a new one is the easiest and safest way to do so.